templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
mhhh… I see we're publishing an article from templates/user/profile.html.
Why not publishing an article from the article's page?